content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -371,6 +371,16 @@ Example: `node.kubernetes.io/pid-pressure:NoSchedule`
 
 The kubelet checks D-value of the size of `/proc/sys/kernel/pid_max` and the PIDs consumed by Kubernetes on a node to get the number of available PIDs that referred to as the `pid.available` metric. The metric is then compared to the corresponding threshold that can be set on the kubelet to determine if the node condition and taint should be added/removed.
 
+### node.kubernetes.io/out-of-service
+
+Example: `node.kubernetes.io/out-of-service:NoExecute`
+
+A user can manually add the taint to a Node marking it out-of-service. If the `NodeOutOfServiceVolumeDetach` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled on
+`kube-controller-manager`, and a Node is marked out-of-service with this taint,
+volume detach operations for Pods terminating on the node will happen immediately without waiting for 6
+minutes timeout period. This allows the Pods on the out-of-service node to recover quickly on a different node. 
+
```

Comment:
Add a note that we require the user to manually remove the out-of-service taint after the pods are moved to a new node and the user has checked that the shutdown node has been recovered since the user was the one who originally added the taint.

Reply:
This should be a brief note. Full docs belong elsewhere.

Reply:
Maybe link to a fragment within https://k8s.io/docs/concepts/architecture/nodes/?